### ISSUE TITLE
Unable to Send SQS message on recent SDK versions

### ISSUE THREAD

**Issue by flavio-cadete-cko** [aws/aws-sdk-net#3114]
### Describe the bug

AWSSDK.SQS 
Until version 3.7.200.67 is working well,
From 3.7.201 or higher I get exceptions running integration tests:
```
Amazon.SQS.AmazonSQSException : The service returned an error with HTTP Body: {}
---- Amazon.Runtime.Internal.HttpErrorResponseException : Exception of type 'Amazon.Runtime.Internal.HttpErrorResponseException' was thrown.
```

### Expected Behavior

To send SQS message

### Current Behavior

Throws exception
```
Amazon.SQS.AmazonSQSException : The service returned an error with HTTP Body: {}
---- Amazon.Runtime.Internal.HttpErrorResponseException : Exception of type 'Amazon.Runtime.Internal.HttpErrorResponseException' was thrown.
```

### Reproduction Steps

Update AWSSDK.SQS to 3.7.201 or higher.
Run integration tests where it uses SQS.
Localstack:  `image: localstack/localstack-light:0.14.5`

### AWS .NET SDK and/or Package version used

`AWSSDK.SQS` `3.7.201` or higher

### Targeted .NET Platform

.NET 6

### Operating System and version

Windows 10

**Comment by ashovlin** [comment]
The AWS SDKs recently switched their protocol for communicating with SQS to JSON to reduce latency and CPU/memory usage. For .NET this is AWSSDK.SQS 3.7.201 or later.
* [Announcement](https://aws.amazon.com/about-aws/whats-new/2023/11/amazon-sqs-support-json-protocol)
* [FAQs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-json-faqs.html)

See https://github.com/localstack/localstack/issues/8267 for the discussion around supporting this from the localstack side. Based on the age of [localstack-light:0.14.5](https://hub.docker.com/layers/localstack/localstack-light/0.14.5/images/sha256-dd3710167293d1a908788c66ceff383d8226f2efa9404f27bdff9926db1cd194?context=explore), I believe that would predate any of the fixes from localstack.

Are you able to upgrade your version of localstack? Otherwise you may need to pin your version of AWSSDK.SQS to a version that is compatible with 0.14.5 as discussed in [What if I am already on the latest AWS SDK version, but my open sourced solution does not support JSON?](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-json-faqs.html#json-protocol-sdk-version-open-source).

**Comment by flavio-cadete-cko** [comment]
Updating localstack from 0.14.x to 1.x requires a lot of changes, I'll pin it for now.
Any urgent fix from AWSSDK.SQS 3.7.201 or above that urge us to make such refactoring?

**Comment by ashovlin** [comment]
1. There have been no SQS-specific changes since the JSON transition in 3.7.201. 
    * 3.7.202 rolled back an breaking change for `ListMessageMoveTasksResultEntry` that was included in 3.7.201. 
    * We took all service packages to 3.7.300 when we [added .NET 8 support](https://github.com/aws/aws-sdk-net/blob/main/changelogs/SDK.CHANGELOG.2023.md#376830-2023-11-10-0208-utc). See that changelog entry for the .NET 8 changes.
    * Beyond that all AWSSDK.SQS versions to date have been due to changes in its dependency AWSSDK.Core, but you can pin newer versions of Core if necessary without updating AWSSDK.SQS.
2. We incremented the 3rd part of the version in this case since it didn't break .NET source or binary compatibility as described in the [versioning section of the README](https://github.com/aws/aws-sdk-net#versioning). SQS itself is compatible with both old and new clients, though we understand that may not be true of 3rd party tools.

**Comment by ashishdhingra** [comment]
@flavio-cadete-cko Please let us know if you need any other inputs from our side. I believe you are following recommendation provided by @ashovlin. Kindly confirm if we are good to close this issue.
